docs/faq/persisting-a-store.md

```diff
@@ -0,0 +1,111 @@
+# Why is my store cleared when I reload the page?
+
+The data in a store, such as [Pinia](https://pinia.vuejs.org/), is only held in memory. There isn't any built-in persistence.
+
+The role of a store is to provide a central place to hold global or application-wide state, rather than keeping it in a specific component. Persisting that state is a separate concern.
+
+But they are often related concerns. If you're persisting data then it typically is 'global', at least in some sense.
+
+We'll discuss various approaches below for persisting data, with or without a store.
+
+## Types of persistence
+
+The first thing to consider is where you want to persist the data. The main options are:
+
+1. On the server, e.g. in a database. This allows the data to be shared between devices or users.
+2. Cookies. While cookies are stored in the client, they are automatically sent as headers on HTTP requests, allowing the server to see the persisted data when handling requests.
+3. Client-side storage, e.g. via the [Web Storage API](https://developer.mozilla.org/en-US/docs/Web/API/Web_Storage_API) (`localStorage`/`sessionStorage`). These won't automatically be included in HTTP requests and have much more generous size limits than cookies. If you're using SSR then you may need to consider how you'll handle rendering on the server, as it won't have access to the persisted data in the user's browser.
+
+Another alternative to consider is 'persisting' the data in the URL. This has other ramifications, such as impacting the browser Back button, but if the requirement is for state to be retained across a browser reload then maybe it should be stored in the page URL? It's usually fairly clear whether this approach is a good fit or not.
+
+For the remainder of this page we're going to focus on client-side persistence, e.g. using cookies, `localStorage` or `sessionStorage`.
```

Comment:
There's nothing down below about cookies, not sure if it's intentionally omitted or it's still WIP (or I'm blind)

Reply:
Ok I'm blind

> It uses `localStorage` by default, or cookies

Reply:
You do make a fair point, cookies only get mentioned in passing.

While I didn't include any specific examples of using cookies, the approach is essentially the same as using `localStorage` or `sessionStorage`, just with a much worse native API.

The VueUse helper for working with cookies is a bit odd. I'm not sure why it's implemented that way, but I would've expected it to expose a cookie value as a ref. Maybe it's not done that way because cookies can be changed by the server, though that isn't really relevant to the use case we're describing here.